### DISCUSSION TITLE
Case suggestions for single board

### DISCUSSION THREAD

**Discussion by brucmod** [tesla-android/issue-tracker#119] [Q&A]
I have been using a great Argon case with heats inks and fan for a while now.  I'm looking for a recommendation on a plastic case with Fan that I can use for this project..

**Comment by adamChrystie** [comment]
My pal is going to print this for me. It fits the low profile ice cooler tower..not the large one.
https://www.thingiverse.com/thing:4911134

For now I have my setup in a 52pi 5 layer acrylic case which can house the larger ice tower cooler fan/heat sink.  Been driving all day, no issues. I think it was $20 USD including the tower cooler fan.
![59CE92B9-73FE-4DF1-9585-81DE1E28D2CF](https://user-images.githubusercontent.com/8594624/199405426-cefc7a6d-131e-4909-85ae-788718899097.jpeg)
![BFA24DF0-67FF-4B8C-9A32-8F0B9DA83EDD](https://user-images.githubusercontent.com/8594624/199405431-c3afc649-5546-4745-8e9a-526abd6305ea.jpeg)
![44B6056E-2FCB-4E1B-8574-542FADCC0623](https://user-images.githubusercontent.com/8594624/199405434-b89f4506-b2d1-4399-8ccd-144a952cee33.jpeg)

**Comment by brucmod** [comment]
I ended up going with 
Miuzei Raspberry Pi 4 Case with Fan Cooling Pi 4 Case with 4 Pcs Aluminum Heat Sinks**
**

![810rG-GoCfL _AC_SL1500_](https://user-images.githubusercontent.com/73072824/199533773-68ed3a06-d6ce-450d-942e-a646d4725ed5.jpg)

**Reply by mikegapinski** [reply]
Waaaaaiiiittttt... This is a piece of ****, that heat sink is a toy. Get something from my hardware requirements list

**Reply by brucmod** [reply]
I canceled my order..  I had the pi4 in an argon one metal case but was worried about signal blockage. So i was looking for alow profile case with fan to replace it 

will this fan work?

https://www.amazon.com/GeeekPi-Raspberry-Cooler-Cooling-Heatsink/dp/B0B9KM9WYL/ref=sr_1_2_sspa?crid=2JINWW7ALUMJP&keywords=ICE+Tower+CPU+Cooling+Fan&sprefix=ice+tower+cpu+cooling+fan%2Caps%2C175&sr=8-2-spons&psc=1

**Reply by mikegapinski** [reply]
Yes, this is the best type of a cooler you can get for a Pi!

**Reply by wyllys66** [reply]
Good to know, I ordered my gear today and went with that GeeekPi heatsink above.

**Reply by brucmod** [reply]
@wyllys66 What case did you go with ?

**Reply by wyllys66** [reply]
The GeekPi one that you linked from Amazon above.

**Reply by philippefraser** [reply]
Hi - did the case work with this fan?
